extensions/src/org/pentaho/platform/web/http/api/resources/services/UserRoleDaoService.java

```diff
@@ -170,7 +202,7 @@ public void setLogicalRoles( LogicalRoleAssignments roleAssignments ) throws Sec
     }
   }
 
-  private boolean canAdminister() {
+  protected boolean canAdminister() {
```

Comment:
This should continue to be a private method, there is no business exposing this to the package just for unit testing when we have viable alternatives using pentaho system.